LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
A non-software license (such as a CC-BY) is needed here to cover non-software content.